docs/learn-about-neptune/team-management.rst

```diff
@@ -135,59 +139,60 @@ Examples are: |credit-default-prediction| and |binary-classification-metrics|.
 
     <a href="https://ui.neptune.ai/neptune-ai/binary-classification-metrics" target="_blank">Binary classification metrics</a>
 
-Create an Organization
+.. _how-to_team-management_create-workspace:
+
+Create a workspace
 ----------------------
-Neptune lets you create an :ref:`organization <core-concepts_organization-types>`.
 
-1. Click the **Context Switcher** at the upper-right corner and click **Create new organization**.
+1. Click the **Context Switcher** at the upper-right corner, then click **Switch workspace**, and finally click **Create new workspace**
```

Comment:
There is no context switcher in upper-right corner anymore. `Click your avatar`?

Reply:
Fixed